proto/internal.pb.go

```diff
@@ -333,6 +333,34 @@ func (m *InternalTruncateLogResponse) Reset()         { *m = InternalTruncateLog
 func (m *InternalTruncateLogResponse) String() string { return proto1.CompactTextString(m) }
 func (*InternalTruncateLogResponse) ProtoMessage()    {}
 
+type InternalChangeReplicasRequest struct {
```

Comment:
They have special-case support for recognizing c, java, and python PB files; I'll file a feature request for `.pb.go`.

https://github.com/github/linguist/pull/471